44.md

```diff
@@ -0,0 +1,167 @@
+NIP-44
+======
+
+Encrypted Payloads (Versioned)
+------------------------------
+
+`optional` `author:paulmillr` `author:staab`
+
+The NIP introduces a new data format for keypair-based encryption. This NIP is versioned to allow multiple algorithm choices to exist simultaneously.
+
+The encrypted payload is encoded as base64. The first byte of the payload represents the version. The rest depends on the version.
+
+Currently defined encryption algorithms:
+
+- `0x00` - Reserved
+- `0x01` - XChaCha with same key `sha256(ecdh)` per conversation
+
+# Version 1
+
+Encoding:
+
+- `1` - 1 byte, version
+- `nonce` - 24 bytes fetched from a crytographically secure random source
+- `ciphertext` - variable number of bytes until the end
+
+`ciphertext` is created using `xchacha(key, nonce, plaintext)`
```

Comment:
xchacha20(...)   (again the 20 should be there)

Reply:
Also at this point you should explain what 'key' is, rather than putting that in a note later, because this is where the thing is being defined.